python/pyhail/dataset.py

```diff
@@ -1258,7 +1626,58 @@ def typecheck(self):
         return self.hc.run_command(self, pargs)
 
     def variant_qc(self):
-        """Compute per-variant QC metrics."""
+        """Compute per-variant QC metrics.
+
+        **Example**
+
+        >>> vds = (hc.read('example.vds')
+        >>>  .variantqc())
+
+        .. _variantqc_annotations:
+
+        **Annotations**
+
+        ``variantqc`` computes 16 variant statistics from the genotype data and stores the results as variant annotations that can be accessed with ``va.qc.<identifier>``:
+
+        +---------------------------+--------+----------------------------------------------------+
+        | Name                      | Type   | Description                                        |
```

Comment:
I've left this as is for now, and let's discuss in check in whether we truly want all annotations in that format. With 16 annotations of various types, all with the same root, I think it's more readable to omit `va.qc` and put in table form.